gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/tasks/ConfigTarTask.java

```diff
@@ -38,9 +37,10 @@ public final AbstractCopyTask from(Object... sourcePaths) {
     }
 
     @Override
-    @SuppressWarnings("RawTypes")
+    @SuppressWarnings({"RawTypes", "deprecation"})
     public final AbstractCopyTask from(Object sourcePath, Closure closure) {
-        return this.from(sourcePath, new ClosureBackedAction<>(closure));
+        // TODO(fwindheuser): Replace usage of 'ClosureBackedAction' before moving to Gradle 8.
```

Comment:
Me, adding this line literally last week

![side_eye_monkey_small](https://user-images.githubusercontent.com/397795/129724564-3cabd852-23eb-45f1-a47e-49e5baaced1e.jpeg)